python/doc/examples/py_graphs/plot_graphs_loglikelihood_contour.py

```diff
@@ -60,14 +60,17 @@ def logLikelihood(X):
     '''
     Evaluate the log-likelihood of a TruncatedNormal on a sample. 
     '''
+    samplesize = sample.getSize()
     mu = X[0]
     sigma = X[1]
     a = sample.getMin()[0]
     b = sample.getMax()[0]
-    distribution = ot.TruncatedNormal(mu,sigma,a,b)
-    samplesize = sample.getSize()
+    delta = (b - a) / samplesize
+    a -= delta
+    b += delta
+    distribution = ot.TruncatedNormal(mu, sigma, a, b)
     samplelogpdf = distribution.computeLogPDF(sample)
-    loglikelihood = -samplelogpdf.computeMean()* samplesize
+    loglikelihood = -samplelogpdf.computeMean() * samplesize
```

Comment:
Then why did you reintroduce the minus sign here?

Reply:
@josephmure I have accepted your proposal using the github interface. I don't know what appended but it get lost, I check this ASAP.